flake.nix

```diff
@@ -0,0 +1,80 @@
+{
+  description = "BizHawk is a multi-system emulator written in C#. BizHawk provides nice features for casual gamers such as full screen, and joypad support in addition to full rerecording and debugging tools for all system cores.";
```

Comment:
Copy from readme instead (the GitHub repo's desc is ancient but I can't edit it).